CHANGELOG.md

```diff
@@ -34,6 +34,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - `getMessageGranules()` - Get the granules from a Cumulus execution message, if any.
   - Added `@cumulus/common/cloudwatch-event/isFailedSfStatus()` to determine if a Step Function status from a Cloudwatch event is a failed status
 
+- **CUMULUS-639**
+  - Adds SAML JWT and launchpad token authentication to Cumulus (configurable)
```

Comment:
```suggestion
  - Adds SAML JWT and launchpad token authentication to Cumulus API (configurable)
```

These changes are just supporting Launchpad for authentication to the Cumulus API, correct?

Reply:
Well, we already had Cumulus API authentication with BYOToken, and I added SAML integration.   I suppose it's Cumulus API (and associated docs that aren't updated.....) Ok.